ambry-api/src/main/java/com.github.ambry/rest/RequestPath.java

```diff
@@ -164,6 +179,13 @@ public String getOperationOrBlobId(boolean stripLeadingSlash) {
     return subResource;
   }
 
+  /**
+   * @return blob segment number for segmented blobs
```

Comment:
add "or NO_BLOB_SEGMENT_IDX_SPECIFIED for full blobs" to the doc

Reply:
added "or NO_BLOB_SEGMENT_IDX_SPECIFIED for non-segmented blobs"